src/current/_includes/cockroachcloud/backups/terraform-managed-backups.md

```diff
@@ -0,0 +1,9 @@
+Set the following for the `backup_config` attribute:
+
+- `enabled` controls whether managed backups are enabled or disabled. If you modify the `retention_days` setting even when managed backups are disabled, this will use the one possible change for `retention_days`. Possible values for the `enabled` setting are: `true` or `false`.
+- `frequency_minutes` determines [how often](#frequency) the managed backup will run in minutes. Possible values are: `5`, `10`, `15`, `30`, `60`, `240` (4 hours), `1440` (24 hours).
+- `retention_days` sets the number of days Cockroach Labs will [retain](#retention) the managed backup in storage. You can change `retention_days` for the cluster **once** (whether in Terraform, the [Cloud API](#cloud-api), or the [Cloud Console](#cloud-console)). Possible values are: `2`, `7`, `30`, `90`, `365`. Note that:
```

Comment:
boy we're really putting a lot of weight on the docs with this "pweez once pweez 🥹 " policy lol